crenel_caves.logic

```diff
@@ -100,16 +100,16 @@ area CrenelCaves:
     node SwitchBridgeNorth:
       door Staircase
       enemy RedChus
-    node SwitchBridgeGap
-	  flag Switch
-	SwitchBridgeNorth -> SwitchBridgeGap: (item Boomerang | item BombBag | item Bow | item CanUseSwordBeam | item CanUsePerilBeam)
-	SwitchBridgeGap -> SwitchBridgeNorth
-	node SwitchBridgeSouth:
+    node SwitchBridgeGap:
+      flag Switch
+    SwitchBridgeNorth -> SwitchBridgeGap: (item Boomerang | item BombBag | item Bow | item CanUseSwordBeam | item CanUsePerilBeam)
+    SwitchBridgeGap -> SwitchBridgeNorth
+    node SwitchBridgeSouth:
       door Doorway
       enemy RedChus
     SwitchBridgeSouth -> SwitchBridgeGap: (item SmithSword | item Boomerang | item BombBag | item Bow)
-	SwitchBridgeGap -> SwitchBridgeSouth
-	SwitchBridgeNorth <-> SwitchBridgeSouth: (item Cape | flag Switch)
+    SwitchBridgeGap -> SwitchBridgeSouth

```

Comment:
The edges away from the `SwitchBridgeGap` node feel unnecessary? I wouldn't have included them, but I guess they don't hurt either?